src/metabase/channel/impl/slack.clj

```diff
@@ -137,7 +137,7 @@
     {:creator  (select-keys ?creator [:first_name :last_name :email :common_name])
      :card     {:id   ?card_id
                 :name ?card_name}
-     :rows     (let [col-names (map :display_name ?result_cols)]
+     :rows     (let [col-names (map :name ?result_cols)]
```

Comment:
Why not display name?

Reply:
it might contain space so it's harder to access. You'll need to use lookup in that casem